conf/httpd.conf

```diff
@@ -50,7 +50,18 @@ TypesConfig /etc/mime.types
 DefaultType text/plain
 MIMEMagicFile conf/magic
 HostnameLookups Off
-ErrorLog /dev/stderr
+<IfDefine NETCAT_LOGS>
+    ErrorLog "|nc localhost 5880"
+    <IfDefine ACCESS_LOGS>
+        TransferLog "|nc localhost 5880"
```

Comment:
i'd feel better if this was `/usr/bin/nc`

Reply:
Done.